prisma/schema.prisma

```diff
@@ -8,66 +8,57 @@ datasource db {
 }
 
```

Comment:
I don't know why it shows that I did changes to this file, cause I didn't, but everything is working.

Reply:
Sorry, I just saw this after submitting my first comment! It looks like there are some unintentional naming changes. My guess is, maybe Prettier updated it? I wonder if we can revert the changes.

Reply:
I don't know what happened, but the only thing I did with the schema was to reference, but not changes, maybe I can resubmit my changes after I change the prisma schema back to what it was before?